lib/spack/spack/mirror.py

```diff
@@ -495,18 +493,18 @@ def error(self):
         self.errors.add(self.current_spec)
 
 
-def add_single_spec(spec, mirror_root, mirror_stats):
+def add_single_spec(spec, mirror, mirror_stats):
```

Comment:
Lately I am wondering if, with the increasing size of the codebase, we should mark functions like this one with a leading underscore to communicate that it is meant to be used only within this module. Not a request, just wondering what do you think about trying to be stricter in specifying the "exported" interface of a package/module.